contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2024-03-21
+title: 'PostHog is now up to 12x cheaper'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* It is now possible to use PostHog's product and web analytics for up to ~12x less, depending on your volumes
+* This change applies to all existing and new customers for any events that have no person properties set
```

Comment:
```suggestion
* This change applies to all customers for any events that don't have person properties
```
If you say 'all' then you don't need the qualifiers 'existing and new'.
Also, 'have no person properties set' is oddly worded, as you're implying the presence of a negative.